index.md

```diff
@@ -1,131 +1,198 @@
 Translations: | [Català](manifesto_for_a_better_world_CA.md) | [Deutsch](manifesto_for_a_better_world_DE.md) | [Français](manifesto_for_a_better_world_FR.md)
 
-The Better World Manifesto
+The Better World Manifesto 
 ==========================
 
+Version 1
```

Comment:
Not sure that's the right place for it, but yes, it has to appear somewhere. Perhaps use _Version 1.0_ (italic + minor version), dunno.